config.json

```diff
@@ -1623,6 +1623,32 @@
         "nugetVersion": "2.2.7",
         "nugetId": "Xamarin.Android.ReactiveX.RxJava",
         "dependencyOnly": true
+      },
+      {
+        "groupId": "com.google.errorprone",
+        "artifactId": "error_prone_annotations",
+        "version": "2.4.0",
+        "nugetVersion": "2.4.0",
+        "nugetId": "Xamarin.Google.ErrorProne.Annotations",
+        "dependencyOnly": false,
+        "templateSet": "errorprone"
+      },
+      {
+        "groupId": "com.google.android",
```

Comment:
I assume this will need to be removed once #546 is merged.

Reply:
true